### DISCUSSION TITLE
2.0.1

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#49] [Announcements]
- build: rollback minSdk to 21
- feat: enable checking for updates
- fix: not able to see comments after seeking to a previous position(Youtube)
- UI: slow regular bullet comments to slightly diminish inconsistent low frame rate on high DPI devices 

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1309/AnimePipe/releases/tag/2.0.1'>2.0.1</a>.</em>

**Comment by Ananas-Kuchen** [comment]
Finally I can update the app. I chose to migrate to this app because it can display replies in the comments column and the UI on the channel group is still the same as the old version (not too big)

Now in the updated version there are separate tabs on channels by category (video, short, live, etc) as well as info. other improvements like bullet comments are working fine.

I saw that someone reported a problem with the live chat. when trying to run some live videos I don't see the chat tab appear at all. hopefully in the next update it will be fixed

Actually if this app can also display the comments column of the live video which will be aired along with showing the countdown time, it will be another plus

Thanks, this is really big update after all 🎉

**Comment by InfinityLoop1308** [comment]
Thanks. Glad to see you enjoyed it.

> I saw that someone reported a problem with the live chat. when trying to run some live videos I don't see the chat tab appear at all. hopefully in the next update it will be fixed.

That is because I am assuming that no meaningful comments will be found in a live. If I am wrong, plz open an issue and attach a screenshot/link to the mistaken resource.

> Actually if this app can also display the comments column of the live video which will be aired along with showing the countdown time, it will be another plus

I never think about this, but it is really a cool idea. Very likely come in next version.
